tools/fv_io.F90

```diff
@@ -419,7 +419,7 @@ subroutine  fv_io_read_restart(fv_domain,Atm)
     Atm(1)%Fv_restart_tile_is_open = open_file(Atm(1)%Fv_restart_tile, fname, "read", fv_domain, is_restart=.true.)
     if (Atm(1)%Fv_restart_tile_is_open) then
       call fv_io_register_restart(Atm(1))
-      call read_restart(Atm(1)%Fv_restart_tile, ignore_checksum=Atm(1)%flagstruct%ignore_rst_cksum)
```

Comment:
May I ask why the ignore_checksum is removed?

Reply:
@mark-a-potts @junwang-noaa  - it looks like the last commit on this branch reverted some changes in fv_io.F90

Reply:
I see it is in the latest dev/emc branch: https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/blob/dev/emc/tools/fv_io.F90#L432

Reply:
I have put the ignore_checksum back in place.